src/subprocess_test.cc

```diff
@@ -261,3 +261,17 @@ TEST_F(SubprocessTest, ReadStdin) {
   ASSERT_EQ(1u, subprocs_.finished_.size());
 }
 #endif  // _WIN32
+
+#ifndef _WIN32
+TEST_F(SubprocessTest, Background) {
+  time_t before = time(NULL);
+  Subprocess* subproc = subprocs_.Add("echo -n 123; sleep 10s&");
```

Comment:
Thanks for adding a test! Will this take 10s to run? If so, is it possible to write a test that doesn't take 10s? (For comparison, running all of ninja's tests takes < 0.5s at the moment and I'd like to keep the test suite fast.)